content/nginx-one/how-to/certificates/manage-certificates.md

```diff
@@ -14,13 +14,22 @@ weight: 100
 
 This guide explains how you can manage SSL/TLS certificates with the F5 NGINX One Console. Valid certificates support encrypted connections between NGINX and your users. 
 
```

Comment:
Put this into a table

- JWT
- Cert and Key for repo
- Cert and Key for actual deployments (for traffic)